### DISCUSSION TITLE
Is it possible to create a server without depending on Bevy?

### DISCUSSION THREAD

**Discussion by gondolyr** [cBournhonesque/lightyear#286] [Q&A] [answered]
I am interested in creating an authoritative server without depending on Bevy so that clients with different implementations can connect to the same server. Something like a TUI client can play the game with a Bevy client. The documentation provides detailed examples of how to create Bevy plugins, though this makes it dependent on using Bevy. Is this possible or is this out of scope for this project? The game I'm envisioning is going to be turn-based.

**Comment by cBournhonesque** [comment, marked as answer]
I guess it's theoretically possible; the server is just listening for some UDP packets, so you just need to make sure that your other non-client sends the correct UDP packets serialized in the correct format to the server! However this project is focused only on the bevy imlpementation
